spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe PostsController do
+
+  describe 'GET index' do
+    it 'assigns all posts as @posts' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.